packaging/flavorgen/flavors/generators.go

```diff
@@ -505,6 +579,45 @@ func newMachineDeployment(cluster clusterv1.Cluster, machineTemplate infrav1.VSp
 	}
 }
 
+func newWindowsMachineDeployment(cluster clusterv1.Cluster, machineTemplate infrav1.VSphereMachineTemplate, bootstrapTemplate bootstrapv1.KubeadmConfigTemplate) clusterv1.MachineDeployment {
+	return clusterv1.MachineDeployment{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: clusterv1.GroupVersion.String(),
+			Kind:       typeToKind(&clusterv1.MachineDeployment{}),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s%s-windows", clusterNameVar, machineDeploymentNameSuffix),
+			Labels:    clusterLabels(),
+			Namespace: namespaceVar,
+		},
+		Spec: clusterv1.MachineDeploymentSpec{
+			ClusterName: clusterNameVar,
+			Replicas:    pointer.Int32Ptr(int32(555)),
```

Comment:
555? whatys that all bout ? :)

Reply:
just a placeholder that would then get replaced with the env var `WINDOWS_WORKER_MACHINE_COUNT`

Reply:
Yup. It's just to make it a valid value before being replaced with a string templating variable in the final output.

To be fair, we wrote this whole flavour generator thing a while ago when there was a lot of boilerplating needed for vsphere clusters, particularly around CSI and CPI. Could potentially get rid of it in a future PR.